CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Beta] Create Solana account automatically on wallet creation or SRP import [#32038](https://github.com/MetaMask/metamask-extension/pull/32038)
 
 ### Fixed
+- Fix `wallet_getCapabilities` with missing networks ([#32237](https://github.com/MetaMask/metamask-extension/pull/32237))
```

Comment:
should be original PR
@matthewwalsh0 sorry to make it complex for you, we can do this 2 ways 
1. I just copied the entry message, you can remove this line and i can append to changelog PR https://github.com/MetaMask/metamask-extension/pull/32172
2. we need to change the PR link as original PR. not the cherrypick one
```suggestion
- Fix `wallet_getCapabilities` with missing networks ([#32238](https://github.com/MetaMask/metamask-extension/pull/32238))
```
let me know which one you prefer

Reply:
The current link is the original PR, I checked what links the other entires in the changelog were using so matched that?

Reply:
my bad